README.md

```diff
@@ -16,6 +16,8 @@ Features:
 	- (Static) Variables (Red)
 	- Engine callback functions (Blue)
 	- (Static) Functions (Green)
+		- Setter functions (Green circle, with an arrow inside it pointing to the right)
```

Comment:
Maybe we can shorten it up, something like:
`- Setter functions (Additional arrow inside it, pointing to the right)`
`- Getter functions (Additional arrow inside it, pointing to the left)`

What do you think?

Reply:
Will you change and force push it? Then I can merge this PR right away